packages/scaffold-config/test/unit/ct-detect-third-party.spec.ts

```diff
@@ -49,6 +49,63 @@ describe('isThirdPartyDefinition', () => {
   })
 })
 
+describe('isRepositoryRoot', () => {
+  beforeEach(async () => {
+    await fs.mkdir('./tmp')
+  })
+
+  afterEach(async () => {
+    await fs.remove('./tmp')
+  })
+
+  it('returns false if there is nothing in the directory', async () => {
+    const isCurrentRepositoryRoot = await isRepositoryRoot(path.resolve('./tmp'))
+
+    expect(isCurrentRepositoryRoot).to.be.false
+  })
+
+  it('returns true if there is a Git directory', async () => {
+    await fs.mkdir('./tmp/.git')
+
+    const isCurrentRepositoryRoot = await isRepositoryRoot(path.resolve('./tmp'))
+
+    expect(isCurrentRepositoryRoot).to.be.true
+  })
+
+  it('returns false if there is a package.json without workspaces field', async () => {
+    await fs.writeFile('./tmp/package.json', `{
```

Comment:
Won't this make a bunch of files in the local repo?

Could use something like

```
path.join(os.tmpdir(), 'some-temp-dir')
```

To make something in `/tmp` - I've seen this pattern elsewhere in the code base.

Reply:
I have beforeEach and afterEach hooks on these tests that create and delete the directory that is created which should clean up any files that are created

Reply:
I still think it's probably better to just use `/tmp` (via `os.tmpdir()`) that's kind of what it's intended for, for the odd case that the files might not be cleaned up (who knows...) 

One benefit would be cross OS compat - eg, does this test run as expected on windows? Probably not, since it's assuming posix when you do `/`, which isn't the case on windows. The core Node.js module `os` handles all this messiness out of the box.

The `os.tmpdir()` is pattern used throughout the code base when we need to create on-the-fly file system fixtures - what do you think? It should be a fairly quick refactor.

Reply:
`os.tmpdir()` sounds like a good idea since `afterEach` is not guaranteed to run.